src/stream_from.cxx

```diff
@@ -38,14 +38,13 @@ constexpr std::string_view class_name{"stream_from"};
 
 
 pqxx::stream_from::stream_from(
-  transaction_base &tx, from_query_t, std::string_view query) :
+  transaction_base &tx, from_query_t, std::string_view query, bool binary /*= false*/) :
         transaction_focus{tx, class_name}, m_char_finder{get_finder(tx)}
 {
-  tx.exec0(internal::concat("COPY ("sv, query, ") TO STDOUT"sv));
+  tx.exec0(binary?internal::concat("COPY ("sv, query, ") TO STDOUT WITH BINARY"sv):internal::concat("COPY ("sv, query, ") TO STDOUT"sv));
```

Comment:
No big deal, since I run automated formatting on the source tree, but please aim to stay below 80 columns.